lib/api/1.1/northbound/nodes.js

```diff
@@ -44,10 +71,15 @@ function nodesRouterFactory (
      */
 
     router.get('/nodes', rest(function (req) {
-        return nodeApiService.getAllNodes(req.query);
-    }, {
-        serializer: 'Serializables.V1.Node',
-        isArray: true
+        return nodeApiService.getAllNodes(req.query)
+        .then(function (nodes){
+            return Promise.map(nodes, function(node) {
+                return _renderNodeObmSettings(node);
+                })
+            .then(function (modifiedNodes){
```

Comment:
@srinia6 is this `.then` block necessary?  It looks like it just returns the fulfillment value from the previous chained promise.